package.json

```
@@ -10,32 +10,36 @@
     "test": "./node_modules/.bin/truffle test",
     "coverage": "./scripts/coverage-verify.sh 100",
     "compile": "./node_modules/.bin/truffle compile",
+
     "migrateToRedT": "./node_modules/.bin/truffle migrate --f 1 --to 3 --network=red-t-identity-admin",
     "initRedT": "./node_modules/.bin/truffle exec ./migrations/4_initialize_contracts.js --network=red-t-first-id",
     "deployAnsRedT": "./node_modules/.bin/truffle migrate --f 5 --to 5 --network=red-t-identity-admin",
     "updateRedT": "./node_modules/.bin/truffle migrate --f 6 --to 6 --network=red-t-identity-admin",
+
     "migrateToRedB": "./node_modules/.bin/truffle migrate --f 1 --to 3 --network=red-b-identity-admin",
     "initRedB": "./node_modules/.bin/truffle exec ./migrations/4_initialize_contracts.js --network=red-b-first-id",
     "deployAnsRedB": "./node_modules/.bin/truffle migrate --f 5 --to 5 --network=red-b-identity-admin",
     "updateRedB": "./node_modules/.bin/truffle migrate --f 6 --to 6 --network=red-b-identity-admin",
+
     "migrateLocal": "./node_modules/.bin/truffle migrate --f 1 --to 3 --network=local-admin",
     "initLocal": "./node_modules/.bin/truffle exec ./migrations/4_initialize_contracts.js --network=local-first-id",
     "deployAnsLocal": "./node_modules/.bin/truffle migrate --f 5 --to 5 --network=local-admin",
     "updateLocal": "./node_modules/.bin/truffle migrate --f 6 --to 6 --network=local-admin",
+
     "lint": "./node_modules/.bin/solium -d contracts/",
     "remixd": "./node_modules/.bin/remixd -s $(pwd)/contracts --remix-ide https://remix.ethereum.org",
     "dockerTestnetRun": "docker run --name geth-node -d -p 8545:8545 ethereum/client-go --dev --rpc",
     "dockerTestnetStop": "docker rm geth-node",
     "dockerTestnetLogs": "docker logs geth-node --tail 40 -f",
```

```suggestion
    "dockerTestnetRun": "docker run --name ethereum-node -d -p 8545:8545 ethereum/client-go --dev --rpc",
    "dockerTestnetStop": "docker rm ethereum-node",
    "dockerTestnetLogs": "docker logs ethereum-node --tail 40 -f",
```
As you modified the container name from `geth-node` to `ethereum-node`, all the commands must be to the same container.

So you can revert the change name, or modify all the other commands.